lib/interceptor.js

```diff
@@ -413,19 +413,18 @@ Interceptor.prototype.query = function query(queries) {
     // Allow all query strings to match this route
     if (queries === true) {
         this.queries = queries;
+        return this;
```

Comment:
@ierceg there is one: `query() with "true" will allow all query strings to pass`. This test was passing because `this.queries` was only modified in `for (var q in queries) {`, but now we are always setting it, so this early return is needed.